src/main/java/seedu/address/model/Entity/PrefixType.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.model.Entity;
+
+public enum PrefixType {
+    P, // Participant
+    M, // Mentor
+    T, // Team
+    I, // Issue
+}
```

Comment:
Hey. This should have a isValid method as well. Exactly like the one in the file you sent me in Telegram.